src/components/DifficultySwitcher/DifficultySwitcher.tsx

```diff
@@ -21,35 +21,37 @@ const DifficultSwitcher: FC<DifficultSwitcherProps> = ({
   difficulty,
   className
 }) => {
-  return (
-    <div className={classnames("difficulty-switcher", className)}>
-      <button
-        className={classnames("difficulty-switcher__button", {
-          "difficulty-switcher__button--selected":
-            difficulty === Difficulty.EASY,
-        })}
-        onClick={() => setDifficulty(Difficulty.EASY)}
-      >
-        EASY
-      </button>
-      <button
+  const levels = [
```

Comment:
Are these being used anywhere? O-o

Reply:
I don't know how to quote my own code but it's used on line 48.